src/mds/Locker.cc

```diff
@@ -2609,6 +2609,12 @@ void Locker::mark_need_snapflush_inode(CInode *in)
   }
 }
 
+bool Locker::is_revoking_any_caps_from(client_t client)
+{
+  auto rlist = revoking_caps_by_client;
+  return true ? rlist.find(client) != rlist.end() : false;
+}
```

Comment:
should be 

bool Locker::is_revoking_any_caps_from(client_t client)
{
auto it = revoking_caps_by_client.find(client);
if (it == revoking_caps_by_client.end())
return false;
return !it->second.empty();
}